const RuntimeException = Jymfony.Component.Testing.Exception.RuntimeException;

/**
 * @memberOf Jymfony.Component.Testing.Exception
 */
export default class DoubleException extends RuntimeException {
}
